pytest/common_lib/shared.py

```diff
@@ -235,37 +302,33 @@ def send_and_await_signature_requests(
             add_deposit=None):
         """
             Sends `num_requests` signature requests and waits for the results.
-        
+
             Each result is processed by the callback function `sig_verification`, which defaults to `assert_signature_success`.
             If a failure is expected, use `assert_signature_failure` instead. Custom callback functions can also be provided.
-        
+
             Args:
                 `num_requests` (int): The number of signature requests to send.
                 `sig_verification` (callable, optional): A callback function to process each signature result.
                     Defaults to `assert_signature_success`.
-        
+
             Returns:
                 None: The function processes the signature results via the callback but does not return a value.
-        
+
             Raises:
                 AssertionError:
                     - If the indexers fail to observe the signature requests before `constants.TIMEOUT` is reached.
                     - If `sig_verification` raisese an AssertionError.
         """
         started = time.time()
-        metrics = [MetricsTracker(node.near_node) for node in self.mpc_nodes]
         tx_hashes, tx_sent = self.generate_and_send_signature_requests(
             num_requests, add_gas, add_deposit)
         print("Sent signature requests, tx_hashes:", tx_hashes)
 
-        self.observe_signature_requests(started, metrics, tx_sent)
```

Comment:
This check is redundant with verifying the transaction results below, and was not implemented correctly anyway. The `mpc_num_signature_requests` metric is cumulative, so the correct thing to do would be to check the metric value before sending the signature requests, then look for the expected updated value.